test/helpers/cons.go

```diff
@@ -148,28 +148,31 @@ const (
 )
 
 var ciliumCLICommands = map[string]string{
-	"cilium endpoint list -o json":    "endpoint_list.txt",
-	"cilium service list -o json":     "service_list.txt",
-	"cilium config":                   "config.txt",
-	"sudo cilium bpf lb list":         "bpf_lb_list.txt",
-	"sudo cilium bpf ct list global":  "bpf_ct_list.txt",
-	"sudo cilium bpf tunnel list":     "bpf_tunnel_list.txt",
-	"cilium policy get":               "policy_get.txt",
-	"cilium status --all-controllers": "status.txt",
-	"sudo cilium debuginfo":           "debuginfo.txt",
+	"cilium endpoint list -o json":          "endpoint_list.txt",
```

Comment:
Hi, 

I do not liked the JSON output here. We should be able to debug endpoints, services as a user will do. The output of `-json` is a bit different, and we should be able to know problems without getting the JSON output. I'm happy if you add a new line without JSON output.

Reply:
I didn't change this, this was just due to a reformatting with `go fmt`. The output of `cilium endpoint list` is contained in the output of `debuginfo.txt`, which should be one of the primary sources for data so we can dogfood it.

Reply:
ok!